app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
```

Comment:
Wrong package naming; according to the current structure `com.azavea.rf.batch.export` as all batch jobs would be launched somehow with Airflow.